packages/vilay/types.ts

```diff
@@ -49,15 +54,40 @@ export const defineVilay = <
     RouteParams?: unknown
     QueryVariables?: Variables
   } = {
-    PageProps: Record<string, never>,
-    RouteParams: Record<string, never>,
+    PageProps: Record<string, never>
+    RouteParams: Record<string, never>
     QueryVariables: Variables
   }
 >(pageExports: {
   PageLayout?: React.FC<PageLayoutProps>
   initRelayEnvironment?: InitRelayEnvironment
   Page?: Page<T['PageProps']>
+  /**
+   * Specify a static object literal for the `<head>` tag content for this page
+   * Overridden by `getPageHead`.
+   */
   head?: HeadExports
```

Comment:
Given that now we have `getPageHead()`, should we keep this static `head`? I personally dislike things like overriding.

Reply:
that's a good idea. agreed, we don't need it for backwards compatibility, it's just more framework overhead